index.html

```diff
@@ -814,5 +814,156 @@ <h4 class="dblue">Languages</h4>
         </li>
       </ul>
     </section>
+    <section id="form-Section">
+      <div id="form-bimg"></div>
+      <p id="form-text" class="Neutrals">
+        I'm always <br />
+        interested in<br />
+        hearing about<br />
+        new projects, so<br />
+        if you'd like to<br />
+        chat please get<br />
+        in touch.
+      </p>
+      <form id="form" action="https://formspree.io/f/xbjbqokr" method="POST">
+        <ul>
+          <li>
+            <ul id="contactInGrid">
+              <li>
+                <div class="inputDiv">
+                  <label for="name"></label>
+                  <input
+                    type="text"
+                    name="name"
+                    maxlength="30"
+                    required="true"
+                    placeholder="Full name"
+                  />
+                </div>
+              </li>
+              <li>
+                <div class="inputDiv">
+                  <label for="email"></label>
+                  <input
+                    id="mail"
+                    type="email"
+                    name="email"
+                    required="true"
+                    placeholder="Email address"
+                  />
+                </div>
+              </li>
+            </ul>
+          </li>
+          <li>
+            <div id="hideBorder" class="inputDiv">
+              <label id="hide" for="Message"></label>
+              <textarea
+                name="message"
+                required="true"
+                id="message"
+                maxlength="500"
+                placeholder="Enter text here"
+                rows="5"
+              ></textarea>
+            </div>
+          </li>
+          <li id="btn-container">
+            <button type="submit" class="see-project">Get in touch</button>
+          </li>
+        </ul>
+      </form>
```

Comment:
- [ ] I think that you're using extra elements in your markup that don't provide the semantic value you think they are: 😱 

The use of lists is not arbitrary, there are some cases in which this makes sense, for example, a list of form elements (which I think was your approach). However, you have another inner list that you're using to lay out your inputs, and doesn't provide any benefit (semantically speaking) to your document. 🤖 

I suggest that you fix this structure and use something like this:


```html
<form>
  <ul>
    <li> <!-- Input one --> </li>
    <li> <!-- Input two --> </li>
    <li> <!-- Text area --> </li>
    <li> <!-- Button --> </li>
  </ul>
</form>
```

This list is referring to the form elements, and actually provide a good semantic value to your page! 🤓 🔧